### PR TITLE
3690 quickboks dev env

### PR DESCRIPTION
**Pull request by emitche**
- Add additional instructions for connecting QuickBooks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,5 +103,9 @@ In order to test any of the quickbook features, you will need to create Intuit D
 6. Select "Accounting" and click "Create app"
 7. Click on "Keys"
 8. Copy the OAuth Key and Secret into your .env file. Use `.env.example` as a template.
+9. While logged into the application (Madeline site), visit Manage > Settings.
+10. Click on the button that says "Connect to Quickbooks."
+11. Sign in to the Quickbooks account for your developer account.
+12. Click authorize to connect your account data to Madeline.
```
Comment (hooverlunch) [inline README.md]: Note that it's better in Markdown to use `1.` for every item in a numbered list like this as it makes adding items in between easier. But this is fine for now.

File: app/controllers/admin/accounting/transactions_controller.rb [hunk 2]
```diff
@@ -7,7 +7,8 @@ def index
 
     @transactions = Accounting::Transaction.all
   rescue
-    flash.now[:error] = 'Error connecting to quickbooks'
+    flash.now[:error] = "Error connecting to QuickBooks. Visit
+      #{view_context.link_to('Settings', admin_settings_path)}.".html_safe
```
Comment (hooverlunch) [inline app/controllers/admin/accounting/transactions_controller.rb]: Wow very nice use of `view_context` :)

Comment (hooverlunch) [inline app/controllers/admin/accounting/transactions_controller.rb]: One thing: you can avoid use of html_safe, I think, if you do like this:

`"Error connecting to QuickBooks. Visit " << view_context.link_to('Settings', admin_settings_path)`

File: app/controllers/admin/loans_controller.rb [hunk 3]
```diff
@@ -170,7 +170,8 @@ def prep_transactions
     rescue Accounting::Quickbooks::FetchError => e
       Rails.logger.error e
       Rails.logger.error e.cause
-      flash.now[:error] = 'Error connecting to quickbooks'
+      flash.now[:error] = "Error connecting to QuickBooks. Visit
+        #{view_context.link_to('Settings', admin_settings_path)}.".html_safe
```
Comment (hooverlunch) [inline app/controllers/admin/loans_controller.rb]: Same thing here with html_safe. Best to avoid it if at all possible.

File: README.md (hunk 1 above)
Comment (linesarefuzzy) [inline README.md]: This is probably because of my use of Typora when editing this file.

**Pull request merged**